src/moveroplot/time_scores.py

```diff
@@ -122,7 +122,12 @@ def _initialize_plots(labels: list):
         nrows=2, ncols=1, tight_layout=True, figsize=(10, 10), dpi=200
     )
     custom_lines = [
-        Line2D([0], [0], color=plot_settings.modelcolors[i], lw=2)
+        Line2D(
+            [0],
+            [0],
+            color=plot_settings.modelcolors[i % len(plot_settings.modelcolors)],
```

Comment:
Looks good to me. But what about further down (L. 240):

model_plot_color = plot_settings.modelcolors[model_idx]

Will it work there even when model_idx becomes larger than plot_settings.modelcolors?